nuxeo-java-client/src/main/java/org/nuxeo/client/api/objects/Document.java

```diff
@@ -278,6 +289,9 @@ public String getName() {
     }
 
     public void set(String key, Object value) {
+        if (value instanceof Calendar || value instanceof Date) {
```

Comment:
But if floats for instance can't work either why allow them? Why limit yourself to only two classes you know don't work when you know only String works?